pkgs/tools/misc/cinc-auditor/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, ruby, bundlerApp, bundlerUpdateScript }:
+
+bundlerApp {
+  pname = "cinc-auditor-bin";
```

Comment:
```suggestion
  pname = "cinc-auditor";
```

Reply:
I think this is not what we want, their site says to use cinc-auditor-bin. https://cinc.sh/start/auditor/

This gem seems to provide the executable.

Reply:
Then we can name it like this. I first thought this is a binray suffix like it is used in arch.